packages/twenty-front/src/modules/auth/sign-in-up/components/SignInUpForm.tsx

```diff
@@ -233,6 +234,8 @@ export const SignInUpForm = () => {
         <ActionLink onClick={handleResetPassword(form.getValues('email'))}>
           Forgot your password?
         </ActionLink>
+      ) : signInUpMode === SignInUpMode.SignIn ? (
+        <ActionLink href={AppPath.SignUp}>Create an account</ActionLink>
       ) : (
         <FooterNote>
           By using Twenty, you agree to the Terms of Service and Data Processing
```

Comment:
@MamathaYarramaneni Seems like terms line should be removed, am I right @Bonapara ?

Reply:
@magrinj it's mentioned that terms should be removed only for the "**SignIn**" page, please correct me, thanks.

Reply:
Yes you are correct @MamathaYarramaneni thanks!

I think one issue is that it does a full refresh instead of a smooth transition through React. This is usually an easy problem to solve but in that particular case it might not be that easy